src/mon/MgrMonitor.cc

```diff
@@ -515,6 +515,24 @@ bool MgrMonitor::prepare_command(MonOpRequestRef op)
     if (changed && pending_map.active_gid == 0) {
       promote_standby();
     }
+  } else if (prefix == "mgr dump") {
+    MgrMap *p = &map;
+    stringstream ds;
+    string format;
+    cmd_getval(g_ceph_context, cmdmap, "format", format, string("plain"));
+    boost::scoped_ptr<Formatter> f(Formatter::create(format));
+    if (f) {
```

Comment:
Increase `mgr dump` print (), more to use habits and consistent with `mon dump` and ` mds dump`